client/src/www/ui_components/app-root.js

```diff
@@ -270,9 +267,12 @@ export class AppRoot extends mixinBehaviors(
 
         paper-toast paper-button {
           color: var(--light-green);
-          text-align: center;
           /* Prevent the button getting too close to the toast's text. */
-          margin-left: 12px;
+          text-align: center;
+        }
+
+        paper-toast paper-button {
+          margin: unset 12px;
```

Comment:
I think we can merge these two selectors? Seems to be identical?

```suggestion
          text-align: center;
          margin: unset 12px;
```